### ISSUE TITLE
Installer needs to check port availability for 8089

### ISSUE THREAD

**Issue by dykstranet** [inblockio/micro-PKC#105]
Installer fails if port is already in use. 

```
Error response from daemon: driver failed programming external connectivity on endpoint micro-pkc-eauth (b61d0676dc3309ae6669a51bc0aec3e8dfc06b350556785d3f0018f3195bd5aa): Bind for 0.0.0.0:8089 failed: port is already allocated
```

Could be other conflicts if docker already installed and running other containers. 

FIX: After shutting down other containers, I was able to install.

Perhaps requirements should mention port 8089 availability?

**Comment by rht** [comment]
Yeah, would be good to add that in the README, that in addition to 8089 for Eauth, 9352 is needed for MediaWiki. Also need to say to edit the `.env` file, where the values of `MEDIAWIKI_PORT` and `EAUTH_PORT` can be customized.

**Comment by dykstranet** [comment]
Roger that. I am preparing a pull request for `README.md'

**Comment by rht** [comment]
Closed via #106.
